README.md

```diff
@@ -18,9 +18,7 @@ The css files are compiled using Sass. When you have finished writing in the `ma
 Alternatively, if you prefer to see the changes as you are writing to the file, you can run `npm run css` to automatically compile the `main.sass` file whenever you make changes as long as the process is running in the terminal.
 
 #### Javascript
-This project uses ReactJs. If you use JSX, make sure to compile it using `npm run parse-jsx --jsfilepre=JSfilename.js`.
-
-After compiling, if want to deploy the app, minify the files using `npm run minify-js --jsfilepre=JSfilename.js --jsfilepre=JSfilename.min.js`. This will create a new `.js` file, so don't forget to correct your html files
+When you want to deploy the app, minify the files using `npm run minify-js --jsfilepre=JSfilename.js --jsfilepre=JSfilename.min.js`. This will create a new `.js` file, so don't forget to correct your html files
```

Comment:
Do we need deployment instructions as well in our README? Perhaps you could move this to it's own `CONTRIBUTING.md` along with the rest of the setup details in this PR itself, and leave the README a blank slate that we can use as a project showcase as originally planned.

Reply:
Alright